### DISCUSSION TITLE
Error Compiling

### DISCUSSION THREAD

**Discussion by Herb3** [Jyers/Marlin#1169] [General Discussion]
I've tried both the Marlin main and bugfixes branch but keep get this error trying to compile using Jyers config files. I can't seem to figure out how to correct this (First time ever trying to compile firmware here)

In file included from Marlin\src\HAL\STM32\../../inc/MarlinConfig.h:49,
                 from Marlin\src\HAL\STM32\MarlinSerial.cpp:21:
Marlin\src\HAL\STM32\../../inc/SanityCheck.h:801:4: error: #error "USE_M73_REMAINING_TIME requires LCD_SET_PROGRESS_MANUALLY"
  801 |   #error "**_USE_M73_REMAINING_TIME requires LCD_SET_PROGRESS_MANUALLY_**"

**Comment by adam3654** [comment]
In my experience, those sanity check errors indicate that a feature is commented out when it shouldn't be. I checked the Jyers configuration_adv.h and both #define's referenced in the sanity check error are not commented. Weird. I would check your copy of the configuration_adv.h around line 1300 and verify the #define's listed in the sanity check are not commented out.

**Reply by Herb3** [reply]
That's what I can't figure out. Only thing I changed was in the config h file (Bed size). In config adv.h nothing was changed. Looks like the problem is in this area but not sure how to fix
![adv_h](https://user-images.githubusercontent.com/87655444/126186917-74a9d584-ca5d-4678-ab5e-bc1523d919fc.jpg)

**Comment by adam3654** [comment]
Wait, are you using Jyers config files with the main marlin source, not Jyers fork of marlin? If so, that probably won't work.

**Reply by Herb3** [reply]
Ahhh that's the problem. . I am using the Marlin bugfix files with the Jyers config. Thanks!

**Reply by Herb3** [reply]
> Wait, are you using Jyers config files with the main marlin source, not Jyers fork of marlin? If so, that probably won't work.

Thanks Adam that was the problem. I was able to successfully compile the firmware once I got on the correct fork. I appreciate the help!

**Reply by adam3654** [reply]
Glad to help.
